angular/index.html

```diff
@@ -4,31 +4,29 @@
 	<head>
 		<meta charset="utf-8">
 
-		<title>JS 404: Intro to AngularJS Workshop for Girl Develop It 2015.</title>
+		<title>GDG - Introduction to AngularJS by Aysegul Yonet</title>
```

Comment:
Hm, can you remove the GDG stuff? You can also remove GirlDevelopIt and JS 404, and simply call it "Intro to AngularJS"